src/Avalonia.Dialogs/Internal/ByteSizeHelper.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Avalonia.Dialogs.Internal
+{
+	public static class ByteSizeHelper
```

Comment:
I think we should make all of the types in that assembly internal. It's all essentially a private API without any compatibility guarantees.

Reply:
👍 the internal folder/namespace isn't really necessary either. Just make the types internal.

Reply:
done